Subject: Linux-RT 2.6.25-rt
From: http://www.kernel.org/pub/linux/kernel/projects/rt/
Acked-by: Sven-Thorsten Dietrich <sdietrich@suse.de>
From leon.woestenberg@gmail.com Mon Mar 24 17:38:54 2008
Date: Fri, 29 Feb 2008 23:11:40 +0100
From: Leon Woestenberg <leon.woestenberg@gmail.com>
To: linux-rt-users@vger.kernel.org, linux-kernel@vger.kernel.org
Cc: Steven Rostedt <rostedt@goodmis.org>
Subject: [PATCH] Fix build,
     missing profile.h include in kernel/sysctl.c (against 2.6.24.3-rt3)

Build fix for kernel/sysctl.c:356: error: 'prof_pid' undeclared here
(not in a function).
The fix is to include <linux/profile.h> which defines prof_pid as external.
Patch is against 2.6.24.3-rt3.

Signed-off-by: Leon Woestenberg <leon@sidebranch.com>

---
 kernel/sysctl.c |    1 +
 1 file changed, 1 insertion(+)

Index: linux-2.6.25-rt1/kernel/sysctl.c
===================================================================
--- linux-2.6.25-rt1.orig/kernel/sysctl.c	2008-05-05 17:32:24.000000000 -0400
+++ linux-2.6.25-rt1/kernel/sysctl.c	2008-05-05 17:32:52.000000000 -0400
@@ -46,6 +46,7 @@
 #include <linux/acpi.h>
 #include <linux/reboot.h>
 #include <linux/ftrace.h>
+#include <linux/profile.h>
 
 #include <asm/uaccess.h>
 #include <asm/processor.h>
